packages/riipen-ui/src/components/Accordion.test.jsx

```diff
@@ -0,0 +1,149 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import Accordion from "./Accordion";
+import AccordionDetails from "./AccordionDetails";
+import AccordionSummary from "./AccordionSummary";
+
+describe("<Accordion>", () => {
+  it("renders without errors", () => {
+    let error;
+    const child = <AccordionDetails />;
+
+    try {
+      mount(<Accordion>{child}</Accordion>);
+    } catch (e) {
+      error = e;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const summary = <AccordionSummary />;
+    const child = <AccordionDetails />;
+
+    const wrapper = mount(
+      <Accordion>
+        {summary}
+        {child}
+      </Accordion>
+    );
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("default props", () => {
+    it("sets correct default props", () => {
+      const child = <AccordionDetails />;
+
+      const wrapper = mount(<Accordion>{child}</Accordion>);
+
```

Comment:
I was able to test the default classes value for Button, does this work for you?

```
 expect(wrapper.find("Accordion").props().classes).toEqual([]);
```